src/main/webapp/about.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
Is the plan to have "about me" content show up on both `about.html` and `user-page.html`?

Reply:
Now that I'm rereading the specs, the "about me" content should show up on just user-page.html. I will delete the about.html!